.github/actions/deploy-backend/action.yml

```diff
@@ -0,0 +1,103 @@
+# action.yml
+name: 'Push to ACR'
+description: 'Build and push to Azure Container Registry. Deploy to function app.'
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: Set environment variables - dev
+      if: github.ref == 'refs/heads/dev-rheft'
```

Comment:
Are we using the branch for test still? Might make sense at some point to use a better name.

Reply:
Agreed. This is what is listed now, but when we have a new test branch, we can  update this single file.